### PR TITLE
Continuous data in right sidebar

### LINKED ISSUE #736: Condense toolbar
image

Each icon and action we add to the top right creates incrementally more pressure to condense the toolbar into a menu, or a series of icons along the top, or both.

### PR DESCRIPTION
**Pull request by colinmegill**
Fixes #736

rightsidebarresize

### CODE REVIEW

**Comment by bkmartinjr** [comment]
Additional review comment: is there a mock showing the direction this is taking?  I really like it, but it raises issues around other requirements (eg, on a small screen, the graph is going to be tiny -- can the user collapse either the right or left sidebars to expand the graph)?   << @liaprins-czi

File: client/src/components/menubar/index.js [hunk 1]
```diff
@@ -378,14 +374,14 @@ class MenuBar extends React.Component {
               dispatch({ type: "increment graph render counter" });
             }}
           >
-            subset to current selection
+            <Icon icon="double-chevron-down" />
```
[the remark this answers is not in the document]
Comment (colinmegill) [reply client/src/components/menubar/index.js]: No, it's not, it's going to be changed for the release. Placeholder.

[the remark this answers is not in the document]
Comment (colinmegill) [reply client/src/components/menubar/index.js]: I don't think we'll forget but we could make an issue.

[the remark this answers is not in the document]
Comment (bkmartinjr) [reply client/src/components/menubar/index.js]: @colinmegill - can you link the issue number, if needed.

**Comment by bkmartinjr** [comment]
@colinmegill - another question - the layout choices seem a bit ad hoc when the window is small.  Here is an extreme example:

image

In particular, this shows:
* the graph entirely disappears
* the LSB/categorical gets small
* the RSB/histograms do not scale

It seems to me that we should have a more adaptive layout, e.g as things get narrow the layout would move to fewer columns, widgets would sale (like the graph), etc.

Here is another, slightly wider example that shows the inconsistency:
image

I'm not sure we need to fix this as part of this PR, but the PR does make the problem worse (ie, the minimum usable width is increased).

CC: @liaprins-czi

**Comment by bkmartinjr** [comment]
@colinmegill - I am also getting a console error on resize (when screen is narrow).

```
Error: <svg> attribute width: A negative value is not valid. ("-14")
eval @ attr.js:19
eval @ each.js:6
eval @ attr.js:55
eval @ setupSVGandBrush.js:17
createToolSVG @ graph.js:413
componentDidUpdate @ graph.js:356
commitLifeCycles @ react-dom.development.js:20064
commitLayoutEffects @ react-dom.development.js:22813
callCallback @ react-dom.development.js:347
invokeGuardedCallbackDev @ react-dom.development.js:397
invokeGuardedCallback @ react-dom.development.js:454
commitRootImpl @ react-dom.development.js:22585
unstable_runWithPriority @ scheduler.development.js:643
runWithPriority$2 @ react-dom.development.js:11305
commitRoot @ react-dom.development.js:22414
runRootCallback @ react-dom.development.js:21554
eval @ react-dom.development.js:11353
unstable_runWithPriority @ scheduler.development.js:643
runWithPriority$2 @ react-dom.development.js:11305
flushSyncCallbackQueueImpl @ react-dom.development.js:11349
flushSyncCallbackQueue @ react-dom.development.js:11338
batchedUpdates$1 @ react-dom.development.js:21648
notify @ Subscription.js:26
notifyNestedSubs @ Subscription.js:68
handleChangeWrapper @ Subscription.js:73
dispatch @ redux.js:229
eval @ index.js:12
eval @ app.js:55
```

**Comment by colinmegill** [comment]
Great points @bkmartinjr. As responsive has been a target but small screens/mobile have not been, I would move to adopt a min-width, perhaps 1240 pixels, which is not ideal but is still usable in the current branch.

**Comment by bkmartinjr** [comment]
@colinmegill - that min-width seems sensible and would resolve the superficial problems.   Thoughts on making that part of this PR?

Also, to confirm, you intend to fix the "all saved" widget location as part of this PR, right?

**Comment by colinmegill** [comment]
Issue filed for min width: #1022

**Pull request merged**